versions/l-/libassuan.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "cf0ffd2dfbb239d338524fc0ea59311a869144c0",
+      "git-tree": "ea749ee9943cd7ff5e409dead9d7ea14f4481315",
+      "version": "2.5.5",
+      "port-version": 0
+    },
+    {
+      "git-tree": "f1b8e54957fd7f1af3ae3dccf5379c7c10fdeb20",
```

```suggestion
      "git-tree": "cf0ffd2dfbb239d338524fc0ea59311a869144c0",
```
Please note that for the json files in _versions/*/*.json_, we should only **add** content.